x-pack/legacy/plugins/ml/public/jobs/new_job_new/recognize/page.tsx

```diff
@@ -134,11 +138,13 @@ export const Page: FC<PageProps> = ({ moduleId, existingGroupIds }) => {
     loadModule();
   }, []);
 
+  /**
+   * Setups recognizer module configuration.
```

Comment:
Possibly change to `Sets up ...`?

Reply:
Fixed in https://github.com/elastic/kibana/pull/49310/commits/d7ea3c628d2d4059974f61ef943aa256b492cafd